server/odc-service/src/main/java/com/oceanbase/odc/service/connection/ConnectionService.java

```diff
@@ -432,34 +454,29 @@ public Map<Long, CheckState> getStatus(@NonNull Set<Long> ids) {
         List<Permission> granted = res2Actions.entrySet().stream()
                 .map(e -> securityManager.getPermissionByActions(e.getKey(), e.getValue()))
                 .collect(Collectors.toList());
+        Set<Long> projectRelatedConnectionIds = databaseService.statsConnectionConfig().stream()
```

Comment:
The `databaseService.statsConnectionConfig()` will return all project related connection ids? This method name and parameters are hard to understand without any relevant information.

Reply:
Yes. Any advice for the method name?

Reply:
btw, `databaseService.statsConnectionConfig()` returns the ConnectionConfigs rather than connectionIds. And I believe `stats` is the exact meaning, the thing that may confuse you is that `statsConnectionConfig` is better placed in `ProjectService`. Will it be better if `statsConnectionConfig` is placed in `ProjectService`?

Reply:
No, users can also gain access to datasources without going through the project.

Reply:
So this is why it should be placed in `ProjectService` rather than like `ConnectionService` or other classes.